packages/components/app/styles/components/button.scss

```diff
@@ -26,6 +26,11 @@ $hds-button-focus-border-width: 3px;
   position: relative;
   width: auto;
 
+  @at-root a#{&} {
+    text-decoration: none;
+    width: fit-content;
+  }
```

Comment:
We have [`button.hds-button` defined further down the page](https://github.com/hashicorp/design-system/pull/220/files#diff-bc76736c4fa05350de917ad2426c66e972a17bf42a5d210fe476f48ba7d800afR319), would it be more consistent (and potentially easier to read) if we define these simply as `a.hds-button`?

Reply:
I see your point here, but I have mixed feelings for this suggestion. Reason is that the one below is a special override that we apply to the component (intended only for the developers, it's a large visual `console.error`, and may also be removed if we this is too much, or risky) while this one is actual styling. I have added a comment above this rule, so it's more obvious what it does.